step.sh

```diff
@@ -1,14 +1,19 @@
 #!/bin/bash
 set -e
 
-SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-BITRISE_STEP_VERSION=$(cat < "$SCRIPT_DIR/package.json" | jq -r '.version')
+# shellcheck disable=SC2154
+if [[ "${fail_safe}" == "true" || "${fail_safe}" == "yes" ]]; then
+  set +e
+fi
 
 # shellcheck disable=SC2154
 if [[ "${debug}" == "true" || "${debug}" == "yes" ]]; then
   set -x
 fi
 
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+BITRISE_STEP_VERSION=$(cat < "$SCRIPT_DIR/package.json" | jq -r '.version')
```

Comment:
Not sure if `jq` comes installed on macos and linux machines, we should probably add it to the step dependencies